src/org/usfirst/frc4904/robot/RobotMap.java

```diff
@@ -132,7 +132,7 @@ public RobotMap() {
 		Component.leftWheelEncoder.setDistancePerPulse(Metrics.WHEEL_PULSES_PER_REVOLUTION);
 		Component.rightWheelEncoder.setDistancePerPulse(Metrics.WHEEL_PULSES_PER_REVOLUTION);
 		Component.chassisDriveMC = new CustomPIDController(0.001, 0.0, -0.002,
-			new EncoderGroup(100, Component.leftWheelEncoder, Component.rightWheelEncoder));
+			new EncoderPair(Component.leftWheelEncoder, Component.rightWheelEncoder, 100, 100));
```

Comment:
If possible, it would be better to go with the default for distanceTolerance and rateTolerance.
I am also guessing that we do not even use them in our code (we can, but we probably do not), so it is almost certainly OK.